test/exports.test.ts

```diff
@@ -67,4 +67,10 @@ describe("exports", () => {
 
     expect(await generate(mod)).toMatchInlineSnapshot('""');
   });
+  it('export function', async () => {
```

Comment:
remove `async`, not using `generate` from `_utils`

Reply:
Looks like I need to add also support for `ProxifiedFunctionDeclaration`: cannot export async functions, magicast unable to resolve: `export sync function x() { return Promise.resovle(42); }`